getting-started.md

```diff
@@ -45,6 +47,10 @@ Here's what these files and directories are:
 * `src` is for source files for your build system. This isn't a convention you have to follow; Horizon doesn't touch anything in this directory.
 * `dist/index.html` is a sample file. You'll replace this as you develop your application, but there's enough in it to verify that Horizon is installed and working.
 * `.hz/config.toml` is a [TOML][] configuration file for the Horizon server.
+* `.hz/schema.toml` is the default schema file. Initially, it's blank.
```

Comment:
Initially `schema.toml` has the default schema which is an admin group and a rule that the admin group has carte blanche to execute any query